<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>805</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Admin Portal</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>key.png</normaloff>key.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="WelcomeLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>791</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Welcome, Administrator</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>791</width>
      <height>471</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Create User</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>60</y>
        <width>81</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Username:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="UserName">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>60</y>
        <width>171</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>60</y>
        <width>81</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Password:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="PassWord">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>60</y>
        <width>171</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="NameofUser">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>160</y>
        <width>171</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>160</y>
        <width>64</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>170</y>
        <width>91</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>User Type:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="userType">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>170</y>
        <width>131</width>
        <height>24</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>student</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>faculty</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>class_advisor</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="Course">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>260</y>
        <width>90</width>
        <height>24</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>CYS</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>CSE</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>AIE</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>260</y>
        <width>64</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Course:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="CreateSubmit">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>340</y>
        <width>104</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Create User</string>
      </property>
     </widget>
     <widget class="QLabel" name="CreateAck">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>300</y>
        <width>831</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QComboBox" name="subjectList">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>220</y>
        <width>131</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="subjectLabel">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>220</y>
        <width>91</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Subject:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Edit User</string>
     </attribute>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>60</y>
        <width>81</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Username:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>120</y>
        <width>81</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Password:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="EditUsername">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>60</y>
        <width>171</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="EditPassword">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>120</y>
        <width>171</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="EditSubmit">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>250</y>
        <width>151</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Change Password</string>
      </property>
     </widget>
     <widget class="QLabel" name="EditAck">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>300</y>
        <width>831</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Delete User</string>
     </attribute>
     <widget class="QPushButton" name="DeleteSubmit">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>250</y>
        <width>151</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Delete User</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>120</y>
        <width>81</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Username:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="DeleteUsername">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>120</y>
        <width>171</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>-7</x>
        <y>150</y>
        <width>811</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;Warning: Do this with full conciousness, &lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;as this will lead to deletion of a user from the database&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="DeleteAck">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>300</y>
        <width>831</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>List Users</string>
     </attribute>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>55</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Username:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="FindUsername">
      <property name="geometry">
       <rect>
        <x>162</x>
        <y>50</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>55</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Name: </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="FindName">
      <property name="geometry">
       <rect>
        <x>502</x>
        <y>50</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>55</y>
        <width>41</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>(or)</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="FindResult">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>210</y>
        <width>431</width>
        <height>192</height>
       </rect>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.0597pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8.0597pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="FindSubmit">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>130</y>
        <width>104</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Find Users</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Notifications</string>
     </attribute>
     <widget class="QTextBrowser" name="notificationsWindow">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>70</y>
        <width>671</width>
        <height>351</height>
       </rect>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.0597pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8.0597pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="NotificationRefresh">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>20</y>
        <width>104</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Refresh</string>
      </property>
     </widget>
     <widget class="QPushButton" name="NotificationClear">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>20</y>
        <width>104</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
